ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/RecordType.java

```diff
@@ -113,6 +113,8 @@
 
     public static final int RESTAT_RECORD = 165;
 
+    public static final int LRA_RECORD = 166;
```

Comment:
I don't think this is used

Reply:
I reserved it. I will need it when we implement recovery.